apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -288,22 +262,17 @@ def write(self, values: pa.Table) -> None:
                     )
                 )
 
-        if self._get_is_sparse():
-            # sparse write
-            with self._tiledb_open("w") as A:
-                A[rowids] = attr_cols_map
-        else:
-            # TODO: This was a quick thing to bootstrap some early ingestion tests but needs more thought.
-            # In particular, rowids needn't be either zero-up or contiguous.
-            assert len(rowids) > 0
-            rowids = sorted(rowids)
-            assert rowids[0] == 0
-
-            # dense write
-            lo = rowids[0]
-            hi = rowids[-1]
-            with self._tiledb_open("w") as A:
-                A[lo : (hi + 1)] = attr_cols_map
+        # TODO: This was a quick thing to bootstrap some early ingestion tests but needs more thought.
+        # In particular, rowids needn't be either zero-up or contiguous.
+        assert len(rowids) > 0
+        rowids = sorted(rowids)
+        assert rowids[0] == 0
```

Comment:
this assert is spurious -- it doesn't allow for writes at other rows - there is nothing magic about rowid == 0